browse/templates/abs/abs.html

```diff
@@ -0,0 +1,185 @@
+{%- extends "base.html" -%}
+
+  {% block title %}[{{ requested_id }}] {{ abs_meta.title }}{% endblock %}
+
+{% block head %}
+  {{ super() -}}
+  {%- if config['LABS_BIBEXPLORER_ENABLED'] -%}
+  <link rel="stylesheet" media="screen" type="text/css" href="/bibex/bibex.css?20181010"/>
+  {%- endif %}
+  <script src="//static.arxiv.org/js/mathjaxToggle.min.js" type="text/javascript"></script>
+  {{- generate_scholar_tags() }}
+{% endblock head %}
+
+{% block body_id %}{% endblock %}
+
+{% block header_class %}{% endblock %}
+{% block header %}
+  <h1><a href="/">arXiv.org</a> &gt; <a href="{{url_for('.list_articles',context=abs_meta.primary_archive.id, subcontext='recent') }}">{{ abs_meta.primary_archive.id }}</a> &gt; arXiv:{{ requested_id }}</h1>
+  <div id="search">
+    <form id="search-arxiv" method="get" action="{{url_for('search_box')}}">
+
+      <div class="wrapper-search-arxiv">
+        <input class="keyword-field" type="text" name="query" placeholder="Search or Article ID" />
+
+        <div class="filter-field">
+          <select name="searchtype">
+            <option value="all" selected="selected">All papers</option>
+            <option value="ti">Titles</option>
+            <option value="au">Authors</option>
+            <option value="abs">Abstracts</option>
+            <option value="ft">Full text</option>
+            <option value="help">Help pages</option>
+          </select>
+        </div>
+        <input class="btn-search-arxiv" value="" type="submit">
+        <div class="links">(<a href="{{url_for('help')}}">Help</a> | <a href="{{url_for('search_advanced')}}">Advanced search</a>)</div>
+      </div>
+    </form>
+  </div>
+{% endblock header %}
+
+{%- block content %}
+{%- include "abs/trackback_rdf.html" %}
+<div id="abs">
+  {% include "abs/extra_services.html" %}
+  <div class="leftcolumn">
+    <div class="subheader">
+      <h1>{% if abs_meta.primary_archive.id != abs_meta.primary_category.id %}{{ abs_meta.primary_archive.name }} > {% endif %}{{ abs_meta.primary_category.name }}</h1>
+    </div>
+    <h1 class="title mathjax"><span class="descriptor">Title:</span>{{ abs_meta.title|tex_to_utf|arxiv_id_urls }}</h1>
+    <div class="authors"><span class="descriptor">Authors:</span>
+      {%- include "abs/author_links.html" %}
+    </div>
+
+    <div class="dateline">{{ generate_dateline() }}</div>
+
+    <blockquote class="abstract mathjax"><span class="descriptor">Abstract:</span>{{ abs_meta.abstract|arxiv_urlize|line_feed_to_br }}</blockquote>
+    <!--CONTEXT-->
+    <div class="metatable">
+      <table summary="Additional metadata">
```

Comment:
Technically this table is a layout table rather than a data table - can we, at some point, replace it with a more accessible layout? Does not need to block beta but bookmarking as a to-do that must get on the list.